website/versioned_docs/version-2.0.0-alpha.72/using-plugins.md

```diff
@@ -154,6 +154,30 @@ module.exports = function (context, options) {
 
 The `my-plugin` folder could also be a fully fledged package with it's own package.json and a `src/index.js` file for example
 
+### Functional definition {#functional-definition}
+
+A functional plugin is like a module plugin except its a function.
+
+Functional plugins have the same contract as module plugins as such they take two parameters : `context` and `options`.
+
+````js title="docusaurus.config.js"
+module.exports = {
+  // ...
+  plugins: [function(contex, options) {
```

Comment:
Add some line-breaks here and. add `// highlight-start` and `// highlight-end` around the plugin function